VisualizationBase/src/items/ViewItem.cpp

```diff
@@ -58,7 +58,10 @@ void ViewItem::initializeForms()
 			->setMajorAxis(GridLayouter::ColumnMajor)
 			->setNodesGetter([](Item* v)
 				{ auto self = static_cast<I*>(v);
-				  return self->nodesGetter(); }));
+				  return self->nodesGetter(); })
+			->setMajorAxisGetter([](Item* v)
```

Comment:
You should remove the call to `setMajorAxis` above, as it will never be used in this case (because you provide a getter)